app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,46 @@
+class ListsController < ApplicationController
+  def new
+    @list = board.lists.build
+  end
+
+  def create
+    @list = board.lists.build(list_params)
```

Comment:
Z tego co czytam to `build` jest aliasem dla `new`. Te metody generalnie robią to samo. http://stackoverflow.com/questions/1253426/what-is-the-difference-between-build-and-new-on-rails